soliket/mflike/mflike.py

```diff
@@ -37,7 +38,7 @@
 
 class MFLike(GaussianLikelihood, InstallableLikelihood):
     _url = "https://portal.nersc.gov/cfs/sobs/users/MFLike_data"
-    _release = "v0.6"
+    _release = "v0.7.1"
```

Comment:
is this correct? shouldn't it be v0.8.3?

Reply:
I left it like in [LAT_MFLike](https://github.com/simonsobs/LAT_MFLike/blob/4fdfb2e08fa747d8f9baf7eda542e57e6dd8876a/mflike/mflike.py#L23), maybe there is no v0.8 folder on nersc

Reply:
Ok! Again, it would be nice to check with PS people. I think it makes sense to update the data versioning to reflect the code versioning